2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_setup.py:_flush():67] Configure stats pid to 58656
2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_setup.py:_flush():67] Loading settings from /home/w2sg-arnav/.config/wandb/settings
2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_setup.py:_flush():67] Loading settings from /home/w2sg-arnav/cotton-disease/notebooks/wandb/settings
2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-04 10:05:06,693 INFO    MainThread:58656 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/w2sg-arnav/cotton-disease/notebooks/wandb/run-20250304_100506-vwr7fb2l/logs/debug.log
2025-03-04 10:05:06,694 INFO    MainThread:58656 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/w2sg-arnav/cotton-disease/notebooks/wandb/run-20250304_100506-vwr7fb2l/logs/debug-internal.log
2025-03-04 10:05:06,694 INFO    MainThread:58656 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x753debee81f0>
2025-03-04 10:05:06,694 INFO    MainThread:58656 [wandb_init.py:init():761] calling init triggers
2025-03-04 10:05:06,694 INFO    MainThread:58656 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-04 10:05:06,694 INFO    MainThread:58656 [wandb_init.py:init():784] starting backend
2025-03-04 10:05:06,695 INFO    MainThread:58656 [wandb_init.py:init():788] sending inform_init request
2025-03-04 10:05:06,699 INFO    MainThread:58656 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-04 10:05:06,700 INFO    MainThread:58656 [wandb_init.py:init():803] backend started and connected
2025-03-04 10:05:06,715 INFO    MainThread:58656 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-04 10:05:06,717 INFO    MainThread:58656 [wandb_init.py:init():896] updated telemetry
2025-03-04 10:05:06,742 INFO    MainThread:58656 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-04 10:05:07,153 ERROR   MainThread:58656 [wandb_init.py:init():1481] error in wandb.init()
Traceback (most recent call last):
  File "/home/w2sg-arnav/anaconda3/envs/cotton_env/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1471, in init
    return wi.init(run_settings, run_config)
  File "/home/w2sg-arnav/anaconda3/envs/cotton_env/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 968, in init
    raise error
wandb.errors.errors.CommError: failed to upsert bucket: returned error 403 Forbidden: {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/w2sg-arnav/cotton-disease/notebooks/wandb/offline-run-20250304_100513-7cya3hdo/logs/debug.log
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/w2sg-arnav/cotton-disease/notebooks/wandb/offline-run-20250304_100513-7cya3hdo/logs/debug-internal.log
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:init():761] calling init triggers
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:init():784] starting backend
2025-03-04 10:05:13,999 INFO    MainThread:58656 [wandb_init.py:init():788] sending inform_init request
2025-03-04 10:05:14,000 INFO    MainThread:58656 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-04 10:05:14,001 INFO    MainThread:58656 [wandb_init.py:init():803] backend started and connected
2025-03-04 10:05:14,005 INFO    MainThread:58656 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-04 10:05:14,006 INFO    MainThread:58656 [wandb_init.py:init():896] updated telemetry
2025-03-04 10:05:14,026 INFO    MainThread:58656 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-04 10:05:14,114 INFO    MainThread:58656 [wandb_init.py:init():995] starting run threads in backend
2025-03-04 10:05:44,261 INFO    MainThread:58656 [wandb_run.py:_console_start():2377] atexit reg
2025-03-04 10:05:44,261 INFO    MainThread:58656 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-04 10:05:44,289 INFO    MainThread:58656 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-04 10:05:44,289 INFO    MainThread:58656 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-04 10:05:44,290 INFO    MainThread:58656 [wandb_init.py:init():1037] run started, returning control to user process
